src/clientdlg.cpp

```diff
@@ -1083,6 +1104,9 @@ void CClientDlg::Connect ( const QString& strSelectedAddress,
         // change connect button text to "disconnect"
         butConnect->setText ( tr ( "D&isconnect" ) );
 
+        // set default recording state
+        MainMixerBoard->SetServerRecordingState( tr( "May be recording" ) );
```

Comment:
What is this "may be recording" thing? Is it or is it not recording? This message just confuses the users.

Reply:
It's explicitly true.

If you connect to a server and that server does not tell you it's recording - because it's too old to do so, that does not mean it's not recording.  It does not mean it is recording.  It means there is no way to know.  It may or may not be.

May be a preference for the indicator ("I don't care to be told").

Reply:
:-)
You can also argument that old client versions do not have the indicator. So it is just a matter of old versions. You need both, a new client and a new server to get that information. But I am fine with that.